recipe_caching/caching_query.py

```diff
@@ -66,17 +66,17 @@ def __iter__(self):
 
     def _get_cache_plus_key(self):
         """Return a cache region plus key."""
-
         dogpile_region = self.cache_regions[self._cache_region.region]
         if self._cache_region.cache_key:
             key = self._cache_region.cache_key
         else:
             key = _key_from_query(self)
+        if getattr(self._cache_region, 'cache_prefix'):
```

Comment:
Would like some :eyes: on this change

1. I changed the behavior, because what was there seemed nonsensical. (The if condition was `if getattr(self, '_cache_region')`, which in this part of the code was trivially true). I changed it to check for whether `cache_prefix` was set on the region. Weird thing is, I don't see any evidence of this attribute ever being set on a cache region, not even in the dogpile.cache source code.
1. was it significant that this was _not_ here before, while it was in fruition?

Reply:
ok, looking at fruition test failures (https://github.com/juiceinc/fruition/pull/1784) I am thinking this needs tweaked, probably to just delete the conditional entirely.

Reply:
alright, I deleted the conditional, which actually makes the code identical *in practice* to the original, which had a conditional that was always True, and this causes the fruition test suite to pass.